components/form_1/form_1.tsx

```diff
@@ -0,0 +1,29 @@
+import { FormikProps } from 'formik';
+import { memo } from 'react';
+import useForm1 from '../../hooks/useForm1';
+import useInitialData from '../../hooks/useInitialdata';
+import style from '../../styles/home.module.scss';
+import { IForm1 } from '../../types/form';
+import NextBtn from '../fields/nextbtn';
+import Form1Fields from './form_1_fields';
+
+function Form1() {
```

Comment:
use arrow functions for creating components, and export in the same line wherever possible.

Reply:
ok sure will do the changes